src/webui/www/private/views/preferences.html

```diff
@@ -2923,8 +2923,8 @@
             let maxSeedingTime = -1;
             if (document.getElementById("maxSeedingTimeCheckbox").checked) {
                 maxSeedingTime = Number(document.getElementById("maxSeedingTimeValue").value);
-                if (Number.isNaN(maxSeedingTime) || (maxSeedingTime < 0) || (maxSeedingTime > 525600)) {
-                    alert("QBT_TR(Seeding time limit must be between 0 and 525600 minutes.)QBT_TR[CONTEXT=HttpServer]");
+                if (Number.isNaN(maxSeedingTime) || (maxSeedingTime < 0)) {
+                    alert("QBT_TR(Seeding time limit must be greater than 0.)QBT_TR[CONTEXT=HttpServer]");
```

Comment:
But zero is allowed.

Reply:
Precisely!
What would be the best way to write this?
Seeding time limit must be equal to or greater than zero. Or, Seeding time limit must have a positive value.

Reply:
> > Or, Seeding time limit must have a positive value.
> 
> 👍

Done.

Reply:
> > > Or, Seeding time limit must have a positive value.
> > 
> > 
> > 👍
> 
> Done.

Seems it is still incorrect.
`Seeding time limit must not have a negative value` is correct one, no?

Reply:
>Seems it is still incorrect.
`Seeding time limit must not have a negative value` is correct one, no?

Sure.